gulpfile.js

```diff
@@ -69,7 +72,30 @@ function updateFrontendWebdriver(cb) {
   // fits the installed chrome version. Not setting the variable will use the latest
   // chromedriver. Note: you'll have to specify the full version number, e.g. 77.0.3865.40
   // which can be found on https://chromedriver.storage.googleapis.com/.
-  const driverVersion = process.env.CHROMEDRIVER_VERSION !== '' ? process.env.CHROMEDRIVER_VERSION : 'latest';
+  var installedVersion;
+  if (process.env.OS === 'Windows_NT') {
+    console.log('Building on Windows');
+    glob.sync('C:\\Program Files (x86)\\Google\\Chrome\\Application\\[0-9]*').forEach(function (file) {
```

Comment:
It's also possible to have an x86 install in which case we'd need to check `C:\\Program Files\\Google...`